src/Joonasw.AspNetCore.SecurityHeaders/HpkpOptions.cs

```diff
@@ -32,6 +33,11 @@ public string HeaderValue
                     value += "; report-uri=\"" + ReportUri + "\"";
                 }
 
+                if (!string.IsNullOrWhiteSpace(ReportTo))
```

Comment:
I haven't seen a reference online that report-to would work with HPKP. Do you have a source which says so? Also, I don't think we should really update HPKP since support for it is being deprecated. People kinda realized it wasn't such a great idea in practice 😅

Reply:
The w3 has a reference in their example supporting it: https://www.w3.org/TR/reporting/#examples

Agreed, if it's being deprecated, we can just leave this alone. I'll pull it.